dashboard/db/migrate/20210402163201_create_scripts_student_resources.rb

```diff
@@ -0,0 +1,17 @@
+class CreateScriptsStudentResources < ActiveRecord::Migration[5.2]
+  def change
+    create_table :scripts_student_resources do |t|
+      t.integer :script_id
+      t.integer :resource_id
+
+      t.index [:script_id, :resource_id], unique: true
+      t.index [:resource_id, :script_id]
```

Comment:
why do we need both these indexes?

Reply:
if you removed the second one, then querying by resource_id alone would be unindexed.

Reply:
then why not just index on resource id?

Reply:
good question! :-) seems like toss-up to me. I think this is just the pattern we've been following.